hw8/uniq.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+import argparse
+import os, sys
+
+# взять аргументы из командной строки
+parser = argparse.ArgumentParser(description = 'Omit repeat lines')
+parser.add_argument('file',
+                    default='',
+                    nargs ='*',
+                    help = 'Path to file. If empty, stdin is used', 
+                    type=str,
+                    action='store')
+
+# сохранить аргументы из командной строки в переменную
+args = parser.parse_args()
+
+# функция для отбора уникальных значений
+def get_unique(data):
+    unique = []
```

Comment:
Для таких штук лучше использовать множества
```suggestion
    unique = set()
```